ipapython/config.py

```diff
@@ -71,7 +70,17 @@ def check_ip_option(option, opt, value):
     try:
         return CheckedIPAddress(value)
     except Exception as e:
-        raise OptionValueError("option %s: invalid IP address %s: %s" % (opt, value, e))
+        raise OptionValueError("option %s: invalid IP address %s: %s" %
+                               (opt, value, e))
+
+def check_ip_option_with_loopback(option, opt, value):
```

Comment:
TBH I think that doing this way is more straightforward. IMHO `functools.partial` adds an unnecessary complexity to the code.